### PR TITLE
contribution guide: add info about TiDB Docs Dash 2024

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

1. Add info about TiDB Docs Dash 2024 to the contribution guide. 
2. Make the detailed steps of creating pull request as a section.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,14 +4,19 @@
 
 ## 我能为 TiDB 文档做什么贡献？
 
-你可以在提升 TiDB 文档质量、易用性、维护效率、翻译效率等方面做贡献，比如：
+🚀 为了给大家带来更好的文档使用体验，我们宣布，首届 TiDB 文档挑战赛 (TiDB Docs Dash) 将于 2024 年 1 月 9 日 - 12 日进行。同时，我们也希望将借此机会将 TiDB 社区成员聚集到一起，为大家带来一次有趣的协作体验。
```
Comment (qiancai) [inline CONTRIBUTING.md:7]:
```suggestion
🚀 为了给大家带来更好的文档使用体验，首届 TiDB 文档挑战赛 (TiDB Docs Dash) 将于 2024 年 1 月 9 日 - 12 日进行。同时，我们也希望将借此机会将 TiDB 社区成员聚集到一起，为大家带来一次有趣的协作体验。
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,14 +4,19 @@
...
+
+- 活动详情和奖励，请参考 [2024 TiDB 文档挑战赛来啦！提 PR、提 Issue 来赢取 Bose 耳机、TiDB 限量款机械键盘等超丰富周边吧](https://asktug.com/t/topic/1019364)。
+- 活动任务列表：<https://asktug.com/t/topic/1019364>
```
Comment (qiancai) [inline CONTRIBUTING.md:10]:
```suggestion
- 活动参与说明：https://github.com/pingcap/docs-cn/issues/15867
https://github.com/pingcap/docs-cn/issues/15867
- 活动任务列表：<https://asktug.com/t/topic/1019364>
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,14 +4,21 @@
 
 ## 我能为 TiDB 文档做什么贡献？
 
-你可以在提升 TiDB 文档质量、易用性、维护效率、翻译效率等方面做贡献，比如：
+🚀 为了给大家带来更好的文档使用体验，首届 TiDB 文档挑战赛 (TiDB Docs Dash) 将于 2024 年 1 月 9 日 - 12 日进行。同时，我们也希望将借此机会将 TiDB 社区成员聚集到一起，为大家带来一次有趣的协作体验。
+
+- 活动详情和奖励，请参考 [2024 TiDB 文档挑战赛来啦！提 PR、提 Issue 来赢取 Bose 耳机、TiDB 限量款机械键盘等超丰富周边吧](https://asktug.com/t/topic/1019364)。
```
Comment (qiancai) [inline CONTRIBUTING.md:9]:
```suggestion
- 活动详情和奖励，请参考 [2024 TiDB 文档挑战赛来啦！提 PR、提 Issue 来赢取 Bose 耳机、TiDB 限量款机械键盘等超丰富周边吧](https://asktug.com/t/topic/1019364)
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -4,14 +4,21 @@
 
 ## 我能为 TiDB 文档做什么贡献？
 
-你可以在提升 TiDB 文档质量、易用性、维护效率、翻译效率等方面做贡献，比如：
+🚀 为了给大家带来更好的文档使用体验，首届 TiDB 文档挑战赛 (TiDB Docs Dash) 将于 2024 年 1 月 9 日 - 12 日进行。同时，我们也希望将借此机会将 TiDB 社区成员聚集到一起，为大家带来一次有趣的协作体验。
+
+- 活动详情和奖励，请参考 [2024 TiDB 文档挑战赛来啦！提 PR、提 Issue 来赢取 Bose 耳机、TiDB 限量款机械键盘等超丰富周边吧](https://asktug.com/t/topic/1019364)
+- 活动参与说明：<https://github.com/pingcap/docs-cn/issues/15867>
+https://github.com/pingcap/docs-cn/issues/15867
```
Comment (qiancai) [inline CONTRIBUTING.md:11]:
```suggestion
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -4,14 +4,20 @@
 
 ## 我能为 TiDB 文档做什么贡献？
 
-你可以在提升 TiDB 文档质量、易用性、维护效率、翻译效率等方面做贡献，比如：
+🚀 为了给大家带来更好的文档使用体验，首届 TiDB 文档挑战赛 (TiDB Docs Dash) 将于 2024 年 1 月 9 日 - 12 日进行。同时，我们也希望将借此机会将 TiDB 社区成员聚集到一起，为大家带来一次有趣的协作体验。
+
+- 活动详情和奖励，请参考 [2024 TiDB 文档挑战赛来啦！提 PR、提 Issue 来赢取 Bose 耳机、TiDB 限量款机械键盘等超丰富周边吧](https://asktug.com/t/topic/1019364)
+- 活动参与说明：<https://github.com/pingcap/docs-cn/issues/15867>
+- 活动任务列表：<https://asktug.com/t/topic/1019364>
```
Comment (qiancai) [inline CONTRIBUTING.md:11]:
```suggestion
- 活动任务列表：<https://github.com/orgs/pingcap/projects/51>
```

Comment (lilin90) [inline CONTRIBUTING.md]:
```suggestion
- 活动任务列表：<https://github.com/orgs/pingcap/projects/51/views/3>
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**